docs/no_numerical_exp_report.tex

```diff
@@ -72,17 +76,86 @@ \section{Method}
 
 
 \section{Configuration}
-The user defines prototypes to be deployed for fuel facilities,
-and reactor deployment scheme.  The reactor deployment causes the demand of fuel
+The user defines prototypes to be deployed for fuel facilities, and the 
+demand equation.  The reactor deployment causes the demand of fuel
```

Comment:
This isn't very clear. Are you trying to say: 
The user defines what prototypes are to be deployed and the positive or negative demand growth equation for one of the commodities. For example, if the user chooses a positive demand growth equation for power, this triggers reactor deployment. The reactor deployment then causes an increase in the demand of fuel, which triggers fuel facility deployment.